tests/Feature/accountControllerTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use Mockery\MockInterface;
+
+class accountControllerTest extends TestCase{
+
+    private array $games;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    public function test_get_wishlist_returns_ok()
+    {
+        $response = $this->get('/wishlist');
+        $response->assertStatus(200);
+    }
+
+    // public function test_get_wishlist_by_id_returns_ok()
```

Comment:
Great job! Only suggestion I might have is to delete the commented out stuff. I tend to forget to do that all the time, haha. Keep up the good work!